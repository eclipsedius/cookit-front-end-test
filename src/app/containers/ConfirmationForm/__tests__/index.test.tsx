import * as React from 'react';
import { ConfirmationForm } from '..';
import { createRenderer } from 'react-test-renderer/shallow';

const shallowRenderer = createRenderer();

describe('<ConfirmationForm />', () => {
  it('should render and match the snapshot', () => {
    const location = { state: { 
        email: 'johndoe@test.com', 
        postalCode: 'XXX XXX' 
    }};
    shallowRenderer.render(<ConfirmationForm location={location}/>);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
