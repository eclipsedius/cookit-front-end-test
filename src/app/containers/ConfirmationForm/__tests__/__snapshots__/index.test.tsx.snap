// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConfirmationForm /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Confirmation
    </title>
  </Helmet>
  <NavBar />
  <ConfirmationForm__Wrapper>
    <h1>
      Your informations
    </h1>
    <h3>
      Your email: 
       
      johndoe@test.com
    </h3>
    <h3>
      Your postal code: 
       
      XXX XXX
    </h3>
  </ConfirmationForm__Wrapper>
</React.Fragment>
`;
